tests/test_win_writer/test_create_win_string.win

```diff
@@ -0,0 +1,49 @@
+mp_grid = 2, 2, 2
+num_iter = 12
+num_wann = 4
+wvfn_formatted = .true.
+
+begin atoms_cart
+ang
+Ga         0.0000000000       0.0000000000       0.0000000000 
+As        -1.4200470448       1.4200470448       1.4200470448 
+end atoms_cart
+
+begin kpoint_path
+G 0.0 0.0 0.0  X 0.5 0.0 0.5
+X 0.5 0.0 0.5  W 0.5 0.25 0.75
+W 0.5 0.25 0.75  K 0.375 0.375 0.75
+K 0.375 0.375 0.75  G 0.0 0.0 0.0
+G 0.0 0.0 0.0  L 0.5 0.5 0.5
+L 0.5 0.5 0.5  U 0.625 0.25 0.625
+U 0.625 0.25 0.625  W 0.5 0.25 0.75
+W 0.5 0.25 0.75  L 0.5 0.5 0.5
+L 0.5 0.5 0.5  K 0.375 0.375 0.75
+U 0.625 0.25 0.625  X 0.5 0.0 0.5
+end kpoint_path
+
+begin kpoints
+      0.0000000000       0.0000000000       0.0000000000
+      0.0000000000       0.0000000000       0.5000000000
+      0.0000000000       0.5000000000       0.0000000000
+      0.0000000000       0.5000000000       0.5000000000
+      0.5000000000       0.0000000000       0.0000000000
+      0.5000000000       0.0000000000       0.5000000000
+      0.5000000000       0.5000000000       0.0000000000
+      0.5000000000       0.5000000000       0.5000000000
+end kpoints
+
+begin projections
+random
+c=     -1.4200470448,      1.4200470448,      1.4200470448:l=     -3.0000000000,mr=      1.0000000000:::r=      1.0000000000:
+c=     -1.4200470448,      1.4200470448,      1.4200470448:l=     -3.0000000000,mr=      2.0000000000:::r=      1.0000000000:
+c=     -1.4200470448,      1.4200470448,      1.4200470448:l=     -3.0000000000,mr=      3.0000000000:::r=      1.0000000000:
+c=     -1.4200470448,      1.4200470448,      1.4200470448:l=     -3.0000000000,mr=      4.0000000000:::r=      1.0000000000:
```

Comment:
Could be that Wannier90 doesn't like the spaces (or `l`, `mr` and `r` as float) that cause the tests to fail.

Reply:
It should be float vs. int (see comment above) - we're also removing spaces, anyway

Reply:
Indeed, it should be real vs float - we are fixing this and in any case removing spaces just to be sure